go.mod

```diff
@@ -86,4 +86,4 @@ replace (
 	k8s.io/klog/v2 => github.com/simonpasquier/klog-gokit/v2 v2.0.1
 )
 
-go 1.15
+go 1.16
```

Comment:
We don't need to bump this. This is for the minimum golang version that Thanos could run with. 
This should even be lowered depending on the features of golang we use.